pkg/argocd/update_test.go

```diff
@@ -1786,7 +1786,7 @@ func Test_CommitUpdates(t *testing.T) {
 		defer cleanup()
 		gitMock.On("Checkout", mock.Anything).Run(func(args mock.Arguments) {
 			args.Assert(t, "mydefaultbranch")
-		}).Return(nil)
+		}).Return(nil).Once()
```

Comment:
Can you please explain why the `.Once()` is necessary?

Reply:
Thank you for your comment.

This is because in the current code, after checking out the source branch, argo cd checks out push branch .

But, technically, this is not necessary.
I'll fix it.

Reply:
Fixed it!